package.json

```diff
@@ -27,18 +27,20 @@
     "grunt-contrib-jshint": "~0.6.0",
     "grunt-contrib-watch": "~0.5.0",
     "grunt-mocha-test": "~0.7.0",
-    "should": "~2.1.0",
-    "mocha": "~1.14.0",
+    "mocha": "~1.18.0",
     "grunt-jsbeautifier": "~0.2.6",
-    "sinon": "1.9.1",
-    "nock": "~7.2.2"
+    "sinon": "1.12.1",
+    "nock": "~7.2.2",
+    "leche": "~2.1.1"
   },
   "dependencies": {
     "async": "~0.2.9",
+    "cache-service": "^1.3.5",
+    "cache-service-node-cache": "^1.1.1",
+    "cache-service-redis": "^1.1.3",
```

Comment:
Not sure such a liberal version policy here is wise - could introduce breaking changes without shrinkwrap